db.go

```diff
@@ -464,7 +480,7 @@ func (x *sanitizingUserStore) ResetPasswordFinish(userId UserId, token string, p
 	return x.backend.ResetPasswordFinish(userId, token, password)
 }
 
-func (x *sanitizingUserStore) CreateIdentity(email, username, firstname, lastname, mobilenumber, password string, authUserType AuthUserType) (UserId, error) {
+func (x *sanitizingUserStore) CreateIdentity(email, username, firstname, lastname, mobilenumber, telephonenumber, remarks string, created int64, createdby string, modified int64, modifiedby, password string, authUserType AuthUserType) (UserId, error) {
```

Comment:
I think all of these function signatures would benefit from passing a struct around, instead the long list of arguments that this has become.